index.html

```diff
@@ -1292,38 +1254,156 @@ <h3>Representation: Ed25519Signature2020</h3>
           </li>
         </ol>
 
-        <pre class="example" title="Signed Credential">
+      <pre class="example" title="Signed Credential"
+      data-include="TestVectors/eddsa-2022/signedDataInt.json" data-include-format="text"></pre>
+    </section>
+    <section>
```

Comment:
Related to indentation issue above, all tags in this section need to go right 2 spaces.

Reply:
Any recommendations for tools to automate this stuff? Hand indentation is quite tedious. I usually just use various auto formatters for my code.

Reply:
I have vim setup in some way that I go top-down to lines with tags that are off and hit '==' and it fixes it.  And I just go through until done.  A bit tedious, but it doesn't take long, usually.  Probably takes as long to write and revise this comment. :-)  I'm not sure what other tools would fix this without causing mass havoc on all the other markup.  With the style in many of these specs you can also grep for tags and see the indentation that way to visually find issues.

Reply:
We used Vim in the '80s at Berkeley. I'll have to try something that works with VS Code...